Makefile

```
@@ -13,9 +13,10 @@ all: build
 .PHONY: overlay
 overlay:
 	$(eval OVERLAY_DIR := $(shell mktemp -d))
-	sudo mount -t overlay overlay -o lowerdir=/:./src:./overlay $(OVERLAY_DIR)
+	sudo mount -t overlay overlay -o lowerdir=/:./src:./overlay:externals/librtui/src $(OVERLAY_DIR)
+	sudo mount -t overlay overlay -o lowerdir=./externals/librtui/src:externals/librtui/examples "$(OVERLAY_DIR)/usr/lib/librtui"
```

```suggestion
	sudo mount -t overlay overlay -o lowerdir=./externals/librtui/src "$(OVERLAY_DIR)/usr/lib/librtui"
```
examples就不用挂载了，路径也不对